examples/IBM.Watson.Assistant.v2.Examples/ServiceExample.cs

```diff
@@ -106,12 +107,15 @@ public void MessageWithContext()
             IamAuthenticator authenticator = new IamAuthenticator(
                 apikey: "{apikey}");
 
-            AssistantService service = new AssistantService("2019-02-28", authenticator);
+            AssistantService service = new AssistantService("2020-04-01", authenticator);
             service.SetServiceUrl("{serviceUrl}");
 
             MessageContextSkills skills = new MessageContextSkills();
             MessageContextSkill skill = new MessageContextSkill();
-            skill.UserDefined.Add("account_number", "123456");
+            Dictionary<string, object> userDefinedDic = new Dictionary<string, object>();

```

Comment:
Can you refactor this to `userDefinedDictionary`?

Reply:
Also, can you sign the CLA?
https://cla-assistant.io/watson-developer-cloud/dotnet-standard-sdk?pullRequest=424

Reply:
Ohh okay, I'll do